hnn_core/network.py

```diff
@@ -484,6 +466,36 @@ def plot(self, ax=None, show=True):
         """
         return plot_spikes_raster(spikes=self, ax=ax, show=show)
 
+    def plot_input(self, ax=None, spike_types=None, show=True):
```

Comment:
Do you think it can be made to work for any cell type? Or is there some fundamental issue to make it work?

Reply:
It should work fine for any cell type! In fact it is hard-coded to ignore spikes from pyr and basket cells [here](https://github.com/jonescompneurolab/hnn-core/blob/b802421711cf4e8e6af3e2ad37a814575a221f65/hnn_core/viz.py#L85). If you remove the lines filtering out non-inputs then `spike_types_mask` will contain an element of every unique gid in `spikes._type`.

Reply:
okay I let you do this :)

Reply:
My current plan is to rename the function in `Spikes` and `hnn_core/viz` to `plot_hist`. However, the default behavior with `spike_types=None` will only plot the inputs (since there is already a raster function?). Let me know if this makes sense or if you have something else in mind. 

It's simple to have everything plotted be default, but the figure gets cluttered really fast.

Reply:
yeah that sounds reasonable to me